### PR TITLE
docs: DLT-1796 move storybook to doc site - Link to Pagination

### PR DESCRIPTION
**Pull request by ninamarina**
# docs: DLT-1796 move storybook to doc site - Link to Pagination

* Link (no changes)
* List Item (updated)
* List item group (added)
* Modal (updated)
* Notice (updated)
* Pagination (updated)

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [ ] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket
[DLT-1796]

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

[DLT-1796]: https://dialpad.atlassian.net/browse/DLT-1796?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/dialtone-documentation/docs/components/modal.md [hunk 1]
```diff
@@ -143,6 +143,8 @@ showHtmlWarning />
 
 ### Fixed header and footer
 
+This is the default behavior that adds the scroll automatically in the modal content and leaves the header and footer fixed.
+
 <code-well-header>
   <example-modal kind="fixed" />
 </code-well-header>
```
Comment (francisrupert) [inline apps/dialtone-documentation/docs/components/modal.md:150]: This variant desperately needs a custom scrollbar. There's zero affordance that it's a scrollable area.

I know that's outside this tasks's scope. Can you log an issue and see if we can get that out sooner? An epic should probably be created to collect all the DT components and recipes that should a custom scrollbar.

image

**Review by francisrupert — APPROVED** [review]
💪

File: apps/dialtone-documentation/docs/components/modal.md (hunk 1 above)
Comment (ninamarina) [reply apps/dialtone-documentation/docs/components/modal.md:150]: I logged the ticket here: https://dialpad.atlassian.net/browse/DLT-1983

Comment (francisrupert) [reply apps/dialtone-documentation/docs/components/modal.md:150]: Thanks, I've chucked it in a [new Epic](https://dialpad.atlassian.net/browse/DLT-1984) alongside other ones I've identified.

File: apps/dialtone-documentation/docs/components/list-item-group.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+title: List item group
+description: The "List Item Group" component uses a non interactive heading which groups list items.
+status: ready
+storybook: https://dialtone.dialpad.com/vue/?path=/story/components-list-item-group--default
+---
+
+<code-well-header>
+  <dt-list-item-group
+    heading="Example Heading"
+    headingClass="d-fw-bold"
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/components/list-item-group.md]: Always use kebab case in the template for consistiency: `heading-class`

File: apps/dialtone-documentation/docs/components/list-item.md [hunk 3]
```diff
@@ -14,11 +14,48 @@ figma_url: https://www.figma.com/file/2adf7JhZOncRyjYiy2joil/DT-Core%3A-Componen
...
+
+The **default** slot contains the main content of the list item.
+
+The **subtitle** slot can be used to display content below the default slot. The slot has smaller text size and lighter color than default slot.
+
+The **bottom** slot can be used to display content below the subtitle slot.
+
+<code-well-header class="d-d-block">
+  <ul>
+    <dt-list-item navigationType="tab">
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/components/list-item.md]: navigation-type

**Review by braddialpad — APPROVED** [review]
Really nice thanks! Just a couple small casing issues.

**Pull request merged**